src/frontend/screens/ObservationEdit/LocationView.tsx

```diff
@@ -19,66 +19,95 @@ const m = defineMessages({
 });
 
 export const LocationView = () => {
-  const liveLocation = useMostAccurateLocationForObservation();
-  const observationValue = usePersistedDraftObservation(
-    observationValueSelector,
+  const observationValue = usePersistedDraftObservation(store => store.value);
+
+  return observationValue?.metadata.manualLocation ? (
+    <LocationViewManualPosition />
+  ) : (
+    <LocationViewUpdatePosition />
+  );
+};
+
+const LocationViewManualPosition = () => {
+  const observationValue = usePersistedDraftObservation(store => store.value);
+  return (
+    <LocationViewInner
+      lat={observationValue?.lat}
+      lon={observationValue?.lon}
+      accuracy={null}
+    />
   );
-  const coordinateFormat = usePersistedSettings(coordinateFormatSelector);
+};
+
+const LocationViewUpdatePosition = () => {
```

Comment:
I haven't read all the surrounding code, but it seems like we are replicating the bug from Mapeo legacy where observation position will update during edits as well as on initial creation? Also it seems future-confusing to have the updates happening within this location view component - it requires someone to remember that if a future UX change decided to not display this component then it would stop the observation from having a saved location. It also means that the observation location is not updated when the user is choosing a category, filling in details, or taking photos.

Reply:
Currently the location View is only mounted if there no `observationId` (aka observations that have yet to be saved to the backend). That avoids the observation being updated if the observation is being edited (as opposed to being created).

In mapeo mobile the location did not update when choosing a category, filling in details, or taking photos. Ir has a similar thing where the `LocationField` updated the location, and that was only [mounted when the observation edit screen was mounted](https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/ObservationEditView.js#L190).

Reply:
Will be addressed in [this](https://github.com/digidem/comapeo-mobile/issues/402)